addons/resource.language.en_gb/resources/strings.po

```diff
@@ -3608,7 +3608,7 @@ msgid "Singles"
 msgstr ""
 
 msgctxt "#1051"
-msgid "Enter web address"
+msgid "Enter Web address"
```

Comment:
I would say it isn´t. "World Wide Web" is a proper name, "web address", "webserver" etc. are not.